src/vis_cpu/wrapper.py

```diff
@@ -0,0 +1,124 @@
+"""
+Simple wrapper for basic usage of vis_cpu.
+"""
+import numpy as np
+from . import vis_cpu, conversions
+
+
+def simulate_vis(ants, fluxes, ra, dec, freqs, lsts, beams, pixel_beams=False, 
+                 beam_npix=63, polarized=False, precision=1, 
+                 latitude=-30.7215*np.pi/180.):
+    """
+    Simple wrapper around `vis_cpu` for running simulations. This wrapper 
+    handles the necessary coordinate conversions etc.
```

Comment:
I think maybe it should be explicitly noted in the docstring that this is an *example* wrapper -- we consider the core `vis_cpu` function to be the simulator engine, and how the user wants to do the co-ordinate transforms etc is up to them.

Reply:
Done.